junction/proposals/templatetags/proposal_filters.py

```diff
@@ -22,3 +22,8 @@ def get_content_urls(proposal):
         return urls
     else:
         return []
+        
+@register.filter(name='has_upvoted_comment')
+def has_upvoted_comment(comment, user):
+     vote = comment.proposalcommentvote_set.get(voter=user)
```

Comment:
`.get` will throw `DoesNotExist` exception if the record isn't found. So you can use filter or handle the exception. I would suggest earlier approach.